tensilelite/Tensile/SolutionStructs.py

```diff
@@ -508,6 +514,8 @@ def __str__(self):
         name += "_Grad%s"%self["DataTypeE"].toChar()
       else:
         name += "_Aux%s"%self["DataTypeE"].toChar() # Not showing aux types
+    if self["OutputAmaxD"]:
+      name += "_AmaxD%s"%self["DataTypeAmaxD"].toChar()
```

Comment:
If DataTypeAmaxD is the default type (== Computetype), we don't need to print the type.

Reply:
Sure, same opinion as the "DataTypeAmaxD" one. It's ok to remove this parameter.

Reply:
done, removed